app/store/reducers/featureFlagsReducer.ts

```diff
@@ -0,0 +1,34 @@
+import { createReducer } from '@reduxjs/toolkit';
+import { FeatureFlagOption } from '../types';
+
+import toggleAllowFeatureFlagsEnabledAction from '../actions/featureFlags/toggleAllowFeatureFlagsEnabledAction';
+import toggleFeatureFlagAction from '../actions/featureFlags/toggleFeatureFlagAction';
+
+const AllFlagsOff: Record<FeatureFlagOption, boolean> = {
+  [FeatureFlagOption.GOOGLE_IMPORT]: false,
```

Comment:
This strikes me as a bit odd, if we are using types on the record here, why do we need have these dynamic keys?

Reply:
I might consider an interface here rather than a record as, a record without all of the keys would be considered valid by the complier but is likely not what we want to be representing.

Reply:
omitting a key throws this error:
<img width="829" alt="image" src="https://user-images.githubusercontent.com/25315679/86062957-cb992380-ba37-11ea-8ef6-24f553aa420e.png">

Is there an additional check that the compiler is missing?

Reply:
TIL, that ts enums are exhaustively checked as keys on records, pretty neat.  I still find this a bit counter intuitive though.